config/application.rb

```diff
@@ -99,11 +99,17 @@ class Application < Rails::Application
     config.active_support.use_authenticated_message_encryption = false
     config.action_dispatch.use_authenticated_cookie_encryption = false
 
-    # Rails 6.0 changed this to :zeitwerk
-    config.autoloader = :zeitwerk
-
     # Rails 6.1 changed this to true, but apparently our codebase is not ready for bidirectional associations
     config.active_record.has_many_inversing = false
+
+    # Rails 7.0 changed this to true
+    config.active_record.verify_foreign_keys_for_fixtures = false
+    config.active_record.automatic_scope_inversing = false
+    # Rails 7.0 changed this to OpenSSL::Digest::SHA256
+    # We'd probably need a rotator:
+    # https://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html#key-generator-digest-class-changing-to-use-sha256
+    config.active_support.hash_digest_class = OpenSSL::Digest::SHA1
```

Comment:
This is tricky one, I've noticed that only when Katello started failing with some hashes not being matched.

Although, this is safe for now, I'd say we need to migrate from SHA1 since RHEL9 won't support it for cert signing purposes at least: https://bugs.ruby-lang.org/issues/19307 
(https://github.com/Katello/katello/pull/11155/files#diff-f6f4727f55ce6e9824ffc740bc9494410dd954416f3bbf50799f38b6738e9fcfL49)
and Rails' defaults to SHA256 for caching already.

If we change this now, we'd probably need that (https://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html#key-generator-digest-class-changing-to-use-sha256) thingy as well as some changes in Katello since I've noticed it heavily utilizes https://github.com/Katello/katello/blob/master/app/lib/katello/util/data.rb#L9 for creating some labels/ids? https://github.com/Katello/katello/blob/master/app/models/katello/content_view_environment.rb#L117, which will break with SHA256. Although, we can fix it in Katello itself.